docs/csharp/local-functions-vs-lambdas.md

```diff
@@ -65,20 +65,20 @@ Third, the compiler can perform static analysis that enables local functions to
 definitely assign captured variables in the enclosing scope. Consider this example:
 
 ```csharp
-bool M()
+int M()
 {
     int y;
-    Local();
+    LocalFunction();
     return y;
 
-    void Local() => y = 0;
+    void LocalFunction() => y = 0;
 }
 ```
 
 The compiler can determine that `Local` definitely assigns `y` when called. Because `Local` is called before the `return` statement, `y` is definitiely
 assigned at the `return` statement.
 
-The analysis that enables that analysis enables the fourth difference.
+The analysis that enables the example analysis enables the fourth difference.
```

Comment:
Even with your clarification of "that", the meaning of this sentence is unclear. I'd suggest removing it, and changing the next sentence to "Fourth, depending..."